app/models/home.js

```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+	jsonBlob: DS.attr('string')
```

Comment:
jsonblob is what type of data it is, but not what the content of the data is. Can you rename with a more meaningful name like pageData.